tls/s2n_async_pkey.h

```diff
@@ -63,8 +63,11 @@ struct s2n_async_pkey_op;
 int s2n_async_pkey_op_perform(struct s2n_async_pkey_op *op, s2n_cert_private_key *key);
 int s2n_async_pkey_op_apply(struct s2n_async_pkey_op *op, struct s2n_connection *conn);
 int s2n_async_pkey_op_free(struct s2n_async_pkey_op *op);
+int s2n_async_pkey_verify_signature(struct s2n_connection *conn, s2n_signature_algorithm sig_alg,
+                                    struct s2n_hash_state *digest, struct s2n_blob *signature);
 
 S2N_RESULT s2n_async_pkey_decrypt(struct s2n_connection *conn, struct s2n_blob *encrypted, struct s2n_blob *init_decrypted,
                            s2n_async_pkey_decrypt_complete on_complete);
```

Comment:
We try to write new methods with S2N_RESULT, and only use int if we have to. Is there a reason you have to use int here?

Reply:
Good point. I reused existing declaration of other api.
Let me move it to S2N_RESULT